docker/entrypoint.sh

```diff
@@ -86,4 +94,4 @@ echo "$INFO Starting $* ..."
 echo "  $SC_USER_NAME rights    : $(id "$SC_USER_NAME")"
 echo "  local dir : $(ls -al)"
 
-su --command "$*" "$SC_USER_NAME"
\ No newline at end of file
+su --command "$*" "$SC_USER_NAME"
```

Comment:
Please have a look at entrypoint.sh in osparc-simcore. Especially since we replaced calls with su to calls with gosu (if I remember correctly because of some issues regarding the processes). I think it would make sense to have the exact same starting procedure.